scripts/bug_retriever.py

```diff
@@ -207,9 +207,55 @@ def retrieve_bugs(self, limit: int | None = None) -> None:
             len(missing_history_bug_ids),
         )
 
+        handel_missing_comments()
```

Comment:
let's do the same for history, as currently we're not retrying downloading them

Reply:
I will do it in a follow-up PR. I filed an issue for that (https://github.com/mozilla/bugbug/pull/4930)